src/systems/filecoin_nodes/repository/ipldstore/_index.md

```diff
@@ -5,25 +5,38 @@ title: "IpldStore - Local Storage for hash-linked data"
 
 {{< readfile file="../../../../libraries/ipld/ipld.id" code="true" lang="go" >}}
 
-Filecoin datastructures are stored in [IPLD](https://ipld.io) format, a data format akin to json built for storage, retrieval and traversal of hash-linked data DAGs.
+IPLD is a set of libraries which allow for the interoperability of content-addressed data structures present in most distributed systems. It provides a fundamental 'common language' to primitive cryptographic hashing, enabling data to be verifiably referenced and retrieved between two independent protocols. For example, a user can reference a git commit in a blockchain transaction to create an immutible copy and timestamp, or a data from a DHT can be refenced and linked to in a smart contract. 
+
+For a more in-depth explanation of IPLD's role in Filecoin, see [this section] of the specification.
```

Comment:
"this section" - which sections do you plan to point to?

Reply:
The section that [this PR](https://github.com/filecoin-project/specs/pull/956) implements, wasn't sure what the link slug would look like ultimately though

Reply:
Ah, let's remove the mention until that lands then.